src/components/Authentication/ResetPassword.js

```diff
@@ -0,0 +1,98 @@
+import React, {useCallback, useState} from "react";
+import {makeStyles} from "@material-ui/core/styles";
+import useAuthStyles from "./styles";
+
+import Button from "@material-ui/core/Button";
+import Grid from "@material-ui/core/Grid";
+import {Link} from "react-router-dom";
+import Paper from "@material-ui/core/Paper";
+import PasswordInput from "./PasswordInput";
+import Typography from "@material-ui/core/Typography";
+
+
+const useStyles = makeStyles((theme) => ({
+    "email": {},
+    "info": {
+        "color": theme.colors.black,
+        "margin-bottom": "10px",
+        "margin-top": "34px",
+    },
+    "requirements": {
+        "color": theme.colors.darkGray,
+        "margin-bottom": "30px",
+        "margin-top": "20px",
+    },
+}));
+
+const ResetPassword = () => {
+    const [password, setPassword] = useState("");
+    const [submitted, setSubmitted] = useState(false);
+    const email = "nelson@summit.com";
```

Comment:
Can we change this to a state var? For testing, we can set "nelson@summit.com" as a default

Reply:
This was a dummy email - when the full reset functionality is implemented, this will be automatically set from the backend/link.